static/js/subs.js

```diff
@@ -366,6 +374,10 @@ exports.mark_sub_unsubscribed = function (sub) {
             settings.collapse('hide');
         }
 
+        // Update subscribers count
+        var count = stream_data.get_subscribers_count(sub.name) - 1;
```

Comment:
Also if you fix this I think you can move the `get_subscribers_count` call into the `rerender_subscribers_count` function too.